package.json

```diff
@@ -7,20 +7,21 @@
     "@material-ui/icons": "^3.0.1",
     "axios": "^0.18.0",
     "classnames": "^2.2.6",
+    "jest": "^23.6.0",
     "lodash": "^4.17.11",
     "moment": "^2.23.0",
     "node-sass": "^4.10.0",
     "prop-types": "^15.6.2",
-    "react": "^16.6.3",
+    "react": "^16.7.0",
```

Comment:
Please revert back to `16.6.3`. If you'd like to upgrade, I'd recommend doing that into a separate PR.